src/test/java/uk/nhs/hee/tis/trainee/reference/dto/validator/CurriculumValidatorTest.java

```diff
@@ -49,6 +49,7 @@ void setUp() {
       "DELETE,MEDICAL_CURRICULUM,false",
       "CURRENT,Other,false",
       "CURRENT,MEDICAL_CURRICULUM,true",
+      "CURRENT,SUB_SPECIALTY, true",
```

Comment:
For the sake of completeness (branch coverage), we should add the other test cases `"INACTIVE,SUB_SPECIALTY,false",...`.